torch/fx/passes/shape_prop.py

```diff
@@ -32,6 +33,10 @@ class TensorMetadata(NamedTuple):
     qparams: dict[str, Any]
 
 
+# When include_contiguity is True, we will set contiguity when its always true for the tensor.
+# some tensors can rerepsent both contiguous and non contiguous tensors. Ex (u0, u1) with (u2, u3).
```

Comment:
. Ex (u0, u1) with (u2, u3).-> ex a tensor of size (u0, u1) and (u2, u3) strides.

Reply:
s/rerepsent/represent